src/STRService/STRService.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <Version>0.0.2-beta.01</Version>
+    <Version>0.0.1-beta</Version>
```

Comment:
this is wrong? we cannot go back to a prev version, did you mean `0.0.2-beta.1`?